aries-site/src/examples/components/table/TableSingleSelectExample.js

```diff
@@ -225,10 +236,54 @@ export const TableSingleSelectExample = () => {
             { property: 'id', header: 'Id', pin: size === 'small' },
             ...columns,
           ]}
-          onClickRow={({ datum }) => onClickHandler(datum)}
+          onClickRow={({ datum }) => setPageDetals(datum)}
           pin={size === 'small'}
         />
       </Box>
     </>
+  ) : (
+    <>
+      <Button
+        onClick={() => {
+          setPageDetals({});
+        }}
+        alignSelf="start"
+        icon={<FormPrevious />}
+        label="Orders"
+      />
+      <Box margin={{ horizontal: 'large' }} border="bottom">
+        <Heading size="small" level={1}>
+          Order Number: {pageDetails.id}
+        </Heading>
+      </Box>
+      <Heading
+        size="small"
+        level={3}
+        margin={{ horizontal: 'large', top: 'large', bottom: 'medium' }}
+      >
+        Details
+      </Heading>
+      <Box
+        margin={{ horizontal: 'large' }}
+        gap="medium"
+        direction="row-responsive"
+      >
+        <Box gap="small" direction="column">
+          {orderDetails.map((item, index) => (
```

Comment:
I think this section should be mapping through and render each item of the `orderDetails` and `pageDetailsList` in a row with `align="start"` so that the line heights match. Right now, it's getting misaligned because the rows have different heights given differing text sizes:
<img width="401" alt="Screen Shot 2021-04-05 at 10 14 36 AM" src="https://user-images.githubusercontent.com/12522275/113610037-75c65d00-9601-11eb-8f82-413953820041.png">